index.bs

```diff
@@ -10200,53 +10249,34 @@ Constructors</h5>
 		the following steps are performed on the <a>rendering thread</a>.
 
 		<div algorithm="AudioWorkletProcessor()">
-			This constructor is invoked by processing a control
-			message queued by the {{AudioWorkletNode()|constructor}}
-			of the associated {{AudioWorkletNode}}. This control
-			message carries the [=processor construction data=] that
-			consists of:
-			<var>nodeName</var>,
-			<var>serializedProcessorPort</var>,
-			<var>serializedOptions</var>,
-			and <var>node</var>.
-
-			If any of these steps throws an exception,
-			abort the rest of steps and
-			<a>queue a task</a> to the <a>control thread</a>
-			<a spec="dom" lt="fire an event">fire</a> an
-			<a href="https://www.w3.org/TR/html50/webappapis.html#the-errorevent-interface">ErrorEvent</a>
-			named <code>processorerror</code> at the associated
-			{{AudioWorkletNode}}.
-
-			1. If there is no [=processor construction data=]
-				passed from the associated {{AudioWorkletNode}},
-				throw a {{TypeError}}.
-
-			1. Let <var>processorPort</var> be
-				[$StructuredDeserializeWithTransfer$](<var>serializedProcessorPort</var>,
-				the current Realm).
-
-			1. Let <var>options</var> be
-				[$StructuredDeserialize$](<var>serializedOptions</var>,
-				the current Realm).
-
-			1. Let <var>processorCtor</var> be the result of looking
-				up <var>nodeName</var> on the
-				{{AudioWorkletGlobalScope}}'s
-				<a>node name to processor constructor map</a>.
-
-			1. Let <var>processor</var> be the result of
-				Construct(<var>processorCtor</var>, « <var>options</var> »).
-
-			1. Set <var>processor</var>’s port to <var>processorPort</var>.
-
+			1. Find <var>[=nodeReference=]</var> that is made
+				available for this constructor. Throw a
+				{{TypeError}} exception if not found.
+			
+			1. If any of following steps throws any exception,
+				abort the rest of steps and <a>queue a task</a>
+				to the <a>control thread</a> to
+				<a spec="dom" lt="fire an event">fire</a> an
+				<a href="https://www.w3.org/TR/html50/webappapis.html#the-errorevent-interface">ErrorEvent</a>
+				named <code>processorerror</code> on
+				<var>nodeReference</var>.
+
+			1. Let <var>processor</var> be
+				<a spec="webidl" lt="this">this</a> value.
```

Comment:
Doesn't `{{this}}` work?  (You don't have to check.)

Reply:
Nope! (just checked)